docs/config.md

```diff
@@ -50,6 +50,14 @@ This setting overrides the `NO_COLOR` environment variable (if set).
 ui.color = "never" # Turn off color
 ```
 
+### Graph style
+
+```toml
+# Possible values: "curved", "square", "ascii-alternative", "ascii-large",
+# "legacy" (default)
```

Comment:
If you're calling the default value "legacy" anyway, maybe rename "ascii-alternative" to "ascii" right away? Then it doesn't have to be renamed later...

Reply:
Makes sense. Done.